frontend/server/src/Controllers/User.php

```diff
@@ -126,29 +126,31 @@ public static function createUser(
         $identity = \OmegaUp\DAO\Identities::findByUsername(
             $createUserParams->username
         );
-        $identityByEmail = \OmegaUp\DAO\Identities::findByEmail(
-            $createUserParams->email
-        );
+        if (isset($createUserParams->email)) {
```

Comment:
```suggestion
        if (!is_null($createUserParams->email)) {
```

we use `is_null` a lot more.